.github/workflows/build.yml

```diff
@@ -91,9 +92,84 @@ jobs:
       - uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
       - uses: jupyterlab/maintainer-tools/.github/actions/check-links@v1
 
+  integration-tests:
+    name: Integration tests
+    needs: build
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        backend: ['local', 'rtc']
+
+    env:
+      PLAYWRIGHT_BROWSERS_PATH: ${{ github.workspace }}/pw-browsers
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+          architecture: 'x64'
+
+      - uses: actions/download-artifact@v4
+        with:
+          name: extension-artifacts
+
+      - name: Install base package
+        run: |
+          set -eux
+          cp ./jupyter_suggestions_core/dist/jupyter_suggestions_core*.whl .
+          pip install "jupyterlab>=4.0.0,<5" jupyter_suggestions_core*.whl
```

Comment:
Hoping we won't be catching too many styling changes from JupyterLab in the next 4.x updates. Otherwise we could narrow the range down a bit.